### ISSUE TITLE
Getting empty plots

### ISSUE THREAD

**Issue by jmartindelasierra** [upsetjs/upsetjs_r#22]
Hello, I always get an empty plot and I can't guess what's the problem as I have limited javascript skills.

I tried both CRAN and developing versions on different platforms (Windows and Ubuntu Linux) and browsers (Chrome, Firefox, R Studio embedded browser...). The examples I try are those in your README.md (simple and Shiny examples). The result is always an empty plot, nothing is displayed.

I'll appreciate any help as I find this visualization and in particular this package very powerful.

Thank you.

**Comment by sgratzl** [comment]
can you post the code that you are using along with which version of R and upsetjs_r you are using?

**Comment by jmartindelasierra** [comment]
This is the code:

```
library(upsetjs)

listInput <-
  list(one = c(1, 2, 3, 5, 7, 8, 11, 12, 13),
       two = c(1, 2, 4, 5, 10),
       three = c(1, 5, 6, 7, 8, 9, 10, 12, 13))

upsetjs() %>%
  fromList(listInput) %>%
  interactiveChart()
```
Without interactivity I get the same result.

R version: 4.1.2
upsetjs version: upsetjs_1.10.10

**Comment by sgratzl** [comment]
can you try the newly released 1.10.11, please

**Comment by jmartindelasierra** [comment]
Thank you @sgratzl, it's working now on my two environments.
